File: questModule/src/main/java/ru/nekit/android/qls/setupWizard/view/QuestSetupWizardFragment.java
package ru.nekit.android.qls.setupWizard.view;

import ru.nekit.android.qls.setupWizard.BaseSetupWizardFragment;
import ru.nekit.android.qls.setupWizard.QuestSetupWizard;


public abstract class QuestSetupWizardFragment extends BaseSetupWizardFragment {

    @Override
    protected QuestSetupWizard getSetupWizard() {
        return (QuestSetupWizard) super.getSetupWizard();
    }

}

File: questPresentation/src/main/java/ru/nekit/android/qls/setupWizard/view/QuestSetupWizardFragment.kt
package ru.nekit.android.qls.setupWizard.view

import android.widget.TextView
import ru.nekit.android.qls.R
import ru.nekit.android.qls.setupWizard.BaseSetupWizardFragment
import ru.nekit.android.qls.setupWizard.QuestSetupWizard

abstract class QuestSetupWizardFragment : BaseSetupWizardFragment() {

    override val setupWizard: QuestSetupWizard
        get() = super.setupWizard as QuestSetupWizard

    val titleView
        get() = view?.findViewById(R.id.tv_title) as TextView

    val descriptionView
        get() = view?.findViewById(R.id.tv_description) as TextView

}
